packages/webapp/src/store/api/types/index.ts

```diff
@@ -115,3 +117,64 @@ export type AnimalRemovalReason = {
   key: AnimalRemovalReasonKeys;
   id: number;
 };
+
+export interface SoilAmendmentMethod {
+  id: number;
+  key: string;
+}
+
+export interface SoilAmendmentPurpose {
+  id: number;
+  key: string;
+}
+
+export interface SoilAmendmentFertiliserType {
+  id: number;
+  key: string;
+}
+
+interface Product {
+  product_id: number | string;
+  name: string;
+  product_translation_key?: string;
+  supplier?: string;
+  type?:
```

Comment:
I think this could be required after LF4260b - we checked prod database and do not have a definition for a product without type